recipe/deploy/update_code.php

```diff
@@ -100,12 +101,20 @@
 
     run("$git remote update 2>&1", ['env' => $env]);
 
+
     // Copy to release_path.
     if (get('update_code_strategy') === 'archive') {
         run("$git archive $targetWithDir | tar -x -f - -C {{release_path}} 2>&1");
-    } else if (get('update_code_strategy') === 'clone') {
+    } else if (get('update_code_strategy') === 'clone' || get('update_code_strategy') === 'remote') {
```

Comment:
Let's just remove `remote` strategy and add

```php
run("$git remote set-url origin $repository", ['env' => $env]);

             $target = get('branch', 'HEAD');
```

to `clone` strategy.

Reply:
Agree! Changed this.